ci/django.yml

```diff
@@ -13,12 +13,12 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.9, 3.11, 3.13]
```

Comment:
:+1: Looks ok; 3.9.20+ through 3.13 are all supported still (`3.7` & `3.8` are EoL)! 

---
⚠️ **Verification**:
  Consider continuous versions or clarify new version choice.

<details><summary>NIT</summary>
@cclauss why `3.9` and `3.11` but no `3.10`, and why skip the default `3.12` in the matrix? This is confusing.

I suggest either:

  * Add both `3.10` and `3.12`:
  ```suggestion
          python-version: ['3.9', '3.10', '3.11', '3.12' '3.13']
  ```
  * Add `3.12` **instead** of 3.9:
  ```suggestion
          python-version: ['3.11', '3.12' '3.13']
  ```
  * Keep `3.9` but add `3.10` Instead of `3.13`:
  ```suggestion
          python-version: ['3.9', '3.10', '3.11']  # 3.12 is already installed by default on ubuntu-latest
  ```
</details>

---

> [!important]
> while perfectly fine for versions pre-`3.10` (e.g. `3.9`), the github yaml parsing seems to prefer quoted values (e.g. [as in python-package.yml](https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml#L19)) than raw Semantic-Versions (e.g. :no_good: `3.11`). If these are the intended versions (see previous consideration), then I'd recommend quoting them for consistency.
```suggestion
        python-version: ["3.9", "3.11", "3.13"]
```

Reply:
The first comment above merely reiterates what is in the URLs in the commit message.

Continuous versions is not an advantage.  Coverage is an advantage.  If tests pass on Py39, Py311, and Py313 then the chances are very high that Py310 and Py312 will pass.  The setup and running of tests on 5 platforms takes more energy and creates more greenhouse gases than 3 for a negligible benefit.  This PR does not want to be more consumptive than the code that it replaces.

If yaml does not need the quotes then yaml does not need the quotes.